app/controllers/snippets_controller.rb

```diff
@@ -49,21 +49,39 @@ def update
     end
   end
 
+  def discuss
+    @snippet = Snippet.find(params[:id])
+    return redirect_to @snippet.slack_url if @snippet.slack_url
```

Comment:
Je mettrais bien un `present?` de sécurité.
```suggestion
    return redirect_to @snippet.slack_url if @snippet.slack_url.present?
```